docs/ABOUT.md

```diff
@@ -1 +1,21 @@
-From web applications to games, the Java language is as diverse as it is portable meaning that it is found nearly everywhere. Some notable examples include Minecraft, Android apps, and large parts of websites like Amazon. With a very strong focus on object oriented programming, it is an ideal language in which to implement complex data structures and algorithms. 
+[Java](https://en.wikipedia.org/wiki/Java_(programming_language)) is a general-purpose computer programming language that is 
+- concurrent, 
+- [class-based](https://en.wikipedia.org/wiki/Class-based_programming), 
+- [object-oriented](https://en.wikipedia.org/wiki/Object-oriented_programming), 
+
+and specifically designed to have as few [implementation dependencies](http://www.java-tips.org/other-api-tips-100035/60-netbeans/1025-what-is-an-implementation-dependency-and-whathowwhen-should-i-use-one.html) as possible. It is intended to let application developers 
+["write once, run anywhere"](https://en.wikipedia.org/wiki/Write_once,_run_anywhere)(WORA) meaning that compiled Java code can run on all platforms that support Java without the need for recompilation. Java applications are typically compiled to [bytecode](https://en.wikipedia.org/wiki/Bytecode) that can run on any [Java virtual machine](https://en.wikipedia.org/wiki/Java_virtual_machine) (JVM) regardless of computer architecture. The language derives much of its syntax from C and C++, but it has fewer low-level facilities than either of them.
+
+Why learn JAVA ?
+
+- JAVA IS EVERYWHERE. It is one of the most in-demand programming languages, a standard for enterprise software, web-based content, games and mobile apps, as well as the Android operating system. 
+
+- Java is Platform Independent: This because of its Write Once, Run Anywhere (WORA) policy.This is still one of the reasons of Java being best programming language, most of Java applications are developed in Windows environment and run in UNIX platform.
```

Comment:
I think you can omit this part about 'Platform Independent', since you addressed it in line 7 above.